tools/souffle_converter.py

```diff
@@ -1067,7 +1067,8 @@ def process_rule(self, rule):
             convertedDisjunctions = [self.convert_terms(x) for x in terms]
             if len(convertedDisjunctions) > 0 and \
                     len(convertedDisjunctions[0]) > 0 and \
-                    convertedDisjunctions[0][0].startswith("not"):
+                    convertedDisjunctions[0][0].startswith("not") or\
```

Comment:
We can do this for now, but in general we will need to reorder the clauses to handle many Souffle programs that we cannot compile today.